### PR TITLE
Add EventHub event publishing feature

### PR DESCRIPTION
**Pull request by binodmx**
Add EventHub event publishing feature for publishing events to Event Hub.

### CODE REVIEW

File: modules/p2-profile/product/pom.xml [hunk 1]
```diff
@@ -3546,7 +3556,7 @@
                                     <id>org.wso2.carbon.metrics.server.feature.group</id>
                                     <version>${carbon.metrics.version}</version>
                                 </feature>
-				            <feature>
+				                <feature>
```
Comment (sdkottegoda) [inline modules/p2-profile/product/pom.xml:3559]: Incorrect indentation?

Comment (binodmx) [reply modules/p2-profile/product/pom.xml:3559]: yes

**Review by sdkottegoda — APPROVED** [review]

**Review by sdkottegoda — APPROVED** [review]

**Comment by sdkottegoda** [comment]
Merging PR with the failure to check jenkins build.

**Review by sdkottegoda — APPROVED** [review]

**Pull request merged**